### PR TITLE
ISPN-10205 Clustered Locks documentation should use EmbeddedClusteredLockManagerFactory

### PR DESCRIPTION
**Pull request by diegolovison**
ISPN-9625 Clustered Locks reliability and consistency XML config

I am reusing the same issue because there are only a few changes. Let me know if you prefer a new one.

### CODE REVIEW

**Comment by karesti** [comment]
@diegolovison Looks good to me, but if you don't mind I rather add a new issue because these changes are not related to the XML configuration of the locks.

File: documentation/src/main/asciidoc/user_guide/clustered_lock.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ include::topics/dependencies/clustered_locks.adoc[]
 
 === ClusteredLock Configuration
 
-Currently there is a single type of `ClusteredLock` supported : non reentrant, NODE ownership lock.
+Currently there are the following `OwnershipLevel`: `NODE` and `INSTANCE`. At this moment, only `NODE` is supported.
```
Comment (danberindei) [inline documentation/src/main/asciidoc/user_guide/clustered_lock.adoc]: "Currently" and "at this moment" mean the same thing :)

Comment (karesti) [reply documentation/src/main/asciidoc/user_guide/clustered_lock.adoc]: I prefer the original doc

Comment (ryanemerson) [reply documentation/src/main/asciidoc/user_guide/clustered_lock.adoc]: The original makes more sense to me too.

Comment (diegolovison) [reply documentation/src/main/asciidoc/user_guide/clustered_lock.adoc]: I am still confused with the original doc. Let me try to explain what I see
```java
public enum OwnershipLevel {
   NODE,
   INSTANCE
}
```
If I open the source code for `ClusteredLock` I saw a lot of methods that won't let me know what are `NODE` or `INSTANCE`.

**Comment by diegolovison** [comment]
Ok. Lets create a new JIRA if the team agree that we should improve the docs :)

File: documentation/src/main/asciidoc/user_guide/clustered_lock.adoc (hunk 1 above)
Comment (karesti) [reply documentation/src/main/asciidoc/user_guide/clustered_lock.adoc]: This is an original error. I added the NODE and INSTANCE, but only node sync is possible today

**Comment by karesti** [comment]
@diegolovison I think we should improve docs, but maybe we should delete the INSTANCE stuff from the code, because it is not supported and won't be supported for a while xD

**Comment by galderz** [comment]
@diegolovison Did you create a new JIRA? You should change the commit message to link to that new JIRA.

**Comment by diegolovison** [comment]
@galderz Done

**Pull request merged**